### ISSUE TITLE
sub queries

### ISSUE THREAD

**Issue by myguidingstar** [walkable-server/walkable#12]
Pull extra data to an entity's keyword with custom sql queries

**Comment by myguidingstar** [comment]
should be on the second position (only after sqb/pull-entities and above p/map-reader and all other plugins) so its data will be available to to other plugins such as derive-attributes and authorization

**Comment by myguidingstar** [comment]
will need to use p/join directly instead of p/join-seq because each entity will issue an sql query

**Comment by myguidingstar** [comment]
sub-selects vs sub-queries?

**Comment by myguidingstar** [comment]
can be batched (aka N+1 problem)?

**Comment by myguidingstar** [comment]
no longer relevant as walkable's internal has changed
